src/fields/Table.php

```diff
@@ -41,6 +41,18 @@ public static function displayName(): string
     // Properties
     // =========================================================================
 
+    /**
+     * @var string|null Custom add row button label
+     */
+    public $addRowLabel = 'Add a row';
```

Comment:
Messages like this should always be run through `Craft::t()`, so the default value can be translated. So leave this value undefined here, and from `init()` add this:

```php
if ($this->addRowLabel === null) {
    $this->addRowLabel = Craft::t('app', 'Add a row');
}
```

Reply:
There should also be a blank line between each of these property definitions.